Common/cpp/Tools/Applier.cpp

```diff
@@ -35,7 +35,10 @@ bool Applier::apply(jsi::Runtime &rt, std::shared_ptr<BaseWorkletModule> module)
     }
     sharedValues.push_back(sv);
   }
-  
+
+  if (this->errorHandler->getError() != nullptr && this->errorHandler->getError()->handled) {
```

Comment:
Why do we need it here?  It should be enough to check this in NativeProxy.

Reply:
fixed 3b7b8a91f96d85c44f230db115b2d07c79cbfcd4

Reply:
ok I checked, this code is needed for iOS, I brought it back in 
3815172
without it on iOS `PreventMultipleEventWorklets` fails in `WorkletFailureTest.js`

Reply:
I think we can move it to the NativeProxy.